scripts/ci/test_source.py

```diff
@@ -62,38 +61,26 @@
                f'ALL_TESTS: {ALL_TESTS}.')
 
 
-class TestExtensionSourceMeta(type):
```

Comment:
The intention of this metaclass is to add test methods to `TestExtensionSource` during the initialization of `TestExtensionSource`.

To verify that, we can add `print(TestExtensionSource.__dict__.keys())` to L97 and check the output:

```
dict_keys(['__module__', 'setUp', 'tearDown', 'test_azext_account', 'test_azext_acrtransfer', 'test_azext_ad', 'test_azext_adp', 'test_azext_aem', 'test_azext_ai_examples', 'test_azext_aks_preview', 'test_azext_alertsmanagement', 
```

It generates a bunch of methods starting with `test_` for each extension so that `unittest` can run them one by one.

Question is: what is the point of it, when it can simply be replaced by a `for` loop and invoke `unittest` or `pytest` on one extension in each iteration?